File: gello_software/ur_rtde/examples/cpp/move_until_contact.cpp
#include <ur_rtde/rtde_control_interface.h>
using namespace ur_rtde;

int main(int argc, char* argv[])
{
  RTDEControlInterface rtde_control("127.0.0.1");

  // Parameters
  std::vector<double> speed = {0, 0, -0.100, 0, 0, 0};
  rtde_control.moveUntilContact(speed);
  rtde_control.stopScript();
  return 0;
}

File: to_delete/gello_software/ur_rtde/examples/py/move_until_contact.py
from rtde_control import RTDEControlInterface as RTDEControl

rtde_c = RTDEControl("127.0.0.1")
speed = [0, 0, -0.100, 0, 0, 0]
rtde_c.moveUntilContact(speed)

rtde_c.stopScript()
